src/index.html

```diff
@@ -1,13 +1,98 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content=
+        "width=device-width, 
+        user-scalable=no, 
+        initial-scale=1.0, 
+        maximum-scale=1.0, 
+        minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="images/imac.jpeg" 
+        alt="product image" 
+        class="card__image"
+      >
+      <div class="card__name">
+        <span class="card__name-text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</span>
+      </div>
+      <div class="card__code">
+        <span class="card__code-text">Product code: 195434</span>
+      </div>
+      <div class="star">
+        <div class="star__stars star__stars--0">
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+        </div>
+    
+        <div class="star__stars star__stars--1">
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+        </div>
+        
+        <div class="star__stars star__stars--2">
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+        </div>
+        
+        <div class="star__stars star__stars--3">
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+        </div>
+        
+        <div class="star__stars star__stars--4">
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+          <div class="star__one-star"></div>
+        </div>
+        
+        <div class="star__stars star__stars--5">
```

Comment:
you can remove all blocks `star__stars` and leave only one with class `star__stars--4`